.aspect/workflows/bazelrc

```diff
@@ -0,0 +1,10 @@
+# build without the bytes
+common --remote_download_outputs=minimal
+common --nobuild_runfile_links
+
+common:rbe --extra_execution_platforms=@aspect_bazel_lib//platforms:x86_64_linux_remote
```

Comment:
I'll be surprised if all the actions in this repo are RBE-friendly, we may have to `no-remote-exec` tag later

Reply:
Seems all current actions pass 100% remote
```

2024-08-28 12:28:01 PDT | (19:28:01) INFO: Found 84 targets and 22 test targets...
-- | --
  | 2024-08-28 12:28:01 PDT | (19:28:01) INFO: Elapsed time: 582.434s, Critical Path: 164.63s
  | 2024-08-28 12:28:01 PDT | (19:28:01) INFO: 1052 processes: 12 remote cache hit, 522 internal, 1 linux-sandbox, 517 remote.
  | 2024-08-28 12:28:01 PDT | (19:28:01) INFO: Build completed successfully, 1052 total actions
```